sdk/spring/spring-cloud-azure-autoconfigure/src/main/java/com/azure/spring/cloud/autoconfigure/jdbc/nativejdbc/CachedTokenCredential.java

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.spring.cloud.autoconfigure.jdbc.nativejdbc;
```

Comment:
what's `nativejdbc`?

Reply:
I put all codes about jdbcplugin without spring level support in this folder.